static/app/views/explore/components/schemaHintsList.tsx

```diff
@@ -103,18 +110,42 @@ function SchemaHintsList({
     return () => resizeObserver.disconnect();
   }, [filterTagsSorted]);
 
+  const onHintClick = useCallback(
+    (hint: Tag) => {
+      if (hint.key === seeFullListTag.key) {
+        return;
+      }
+
+      const newSearchQuery = new MutableSearch(exploreQuery);
+
+      newSearchQuery.addFilterValue(
+        hint.key,
+        hint.kind === FieldKind.MEASUREMENT ? '>0' : ''
+      );
+      setExploreQuery(newSearchQuery.formatString());
+    },
+    [exploreQuery, setExploreQuery]
+  );
+
   const getHintText = (hint: Tag) => {
-    if (hint.key === 'seeFullList') {
-      return t('See full list');
+    if (hint.key === seeFullListTag.key) {
+      return hint.name;
     }
 
-    return tct('[tag] is ...', {tag: prettifyTagKey(hint.key)});
+    return tct('[tag] [operation] ...', {
+      tag: prettifyTagKey(hint.name),
+      operation: hint.kind === FieldKind.MEASUREMENT ? '>' : 'is',
+    });
```

Comment:
Shouldn't localize this since `tag` is something defined in the users code and I think `is` is a keyword used in the search syntax